tidb-cloud/migrate-incremental-data-from-mysql.md

```diff
@@ -0,0 +1,245 @@
+  route-rule-1:               # Rule name.
+    schema-pattern: "tpcc"    # Rule for matching upstream schema names. It supports the wildcards "*" and "?".
+    target-schema: "tpdd"     # Name of the target schema.
+
+filters:
+  tpcc-filter-rule:
+    schema-pattern: "tpcc"
+    events: ["drop database"]
+    action: Ignore
+
+##  ******** Ignore check items **********
+ignore-checking-items: ["table_schema"]
+```
+
+For detailed task configurations, see [DM Task Configurations](https://docs.pingcap.com/tidb/stable/task-configuration-file-full).
+
+To run a data migration task smoothly, DM triggers a precheck automatically at the start of the task and returns the check results. DM starts the migration only after the precheck is passed. To trigger a precheck manually, run the check-task command.
+
+```shell
+[root@localhost ~]# tiup dmctl --master-addr 172.16.7.140:9261 check-task dm-task1.yaml
+tiup is checking updates for component dmctl ...
+Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 172.16.7.140:9261 check-task dm-task1.yaml
+{
+    "result": true,
+    "msg": "check pass!!!"
+}
+```
+
+## Step 4. Start the migration task
+
+Run the following command to start the migration task:
+
+```shell
+[root@localhost ~]# tiup dmctl --master-addr 172.16.7.140:9261 start-task dm-task1.yaml
+tiup is checking updates for component dmctl ...
+Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 172.16.7.140:9261 start-task dm-task1.yaml
+{
+    "result": true,
+    "msg": "",
+    "sources": [
+        {
+           "result": true,
+            "msg": "",
+            "source": "mysql-replica-01",
+            "worker": "dm-172.16.7.154-9262"
+        }
+    ],
+    "checkResult": ""
+}
+```
+
+The parameters used in the command above are described as follows:
+
+|Parameter              |Description    |
+|-                      |-              |
+|`--master-addr`        |The {advertise-addr} of any DM-master in the cluster where `dmctl` is to be connected, e.g.: 172.16.10.71:8261|
+|`start-task`           |Starts the migration task.|
+
+If the task fails to start, check the prompt message and fix the configuration. After that, you can re-run the command above to start the task.
+
+If you encounter any problem, refer to [DM error handling](https://docs.pingcap.com/tidb/stable/dm-error-handling) and [DM FAQ](https://docs.pingcap.com/tidb/stable/dm-faq).
+
+## Step 5. Check the migration task status
+
+To learn whether the DM cluster has an ongoing migration task and view the task status, run the `query-status` command using `tiup dmctl`:
+
+```shell
+[root@localhost ~]# tiup dmctl --master-addr 172.16.7.140:9261 query-status test-task1
+tiup is checking updates for component dmctl ...
```

Comment:
Please separate the output from the command.

Reply:
Done